src/main/java/nl/hsac/fitnesse/fixture/slim/web/NgBrowserTest.java

```diff
@@ -93,7 +96,7 @@ protected void waitForAngularIfNeeded(Method method) {
      */
     protected boolean requiresWaitForAngular(Method method) {
         String methodName = method.getName();
-        return !METHODS_NO_WAIT.contains(methodName);
+        return !methodsNoWait.contains(methodName);
```

Comment:
Why does do all methods have to be inspected in the constructor, instead of checking for the annotation in this method, after checking the static list (possibly caching the result in a field so it does not have to be repeated on each call)? That way not all methods have to be checked for each instance created, which is quite expensive time wise. And I expect most methods will not be annotated. Furthermore if the annotation is NOT present a wait was required, so the time spent checking for it is most likely not that important.